src/control/SConscript

```diff
@@ -129,14 +129,20 @@ def scons():
     nvmecontrol = denv.StaticLibrary(
         "nvme_control",
         join(gospdkpath, "src", "nvme_control.c"),
-        CC=compiler, LIBS=['spdk'])
+        CC=compiler)
 
     denv.Install(join(denv.subst("$PREFIX"), "lib"), nvmecontrol)
 
     # CGO shell env vars.
     denv.AppendENVPath(
         "CGO_LDFLAGS",
-        denv.subst("-L%s -L$SPDK_PREFIX/lib $_RPATH" % gopath))
+        denv.subst("-L%s -L$SPDK_PREFIX/lib -L$PREFIX/opt/isal/lib "
```

Comment:
I'm not sure why isal lib can't be found in my env unless I specify it here. Tom, do you know if there is any better place to specify this?

Reply:
yes you shouldn't need to specify explicitly, also I'm not sure /opt should be specified in path, @jolivier23 have any ideas ?